.github/workflows/test-linux.yml

```
@@ -20,4 +20,4 @@ jobs:
       - name: Build
         run: make build-ci -j2 V=1
       - name: Test
-        run: make run-ci -j2 V=1
+        run: make run-ci -j2 V=1 TEST_CI_PROGRESS="-p dots"
```

You should be able to do this:
```suggestion
        run: make run-ci -j2 V=1 TEST_CI_ARGS="-p dots"
```

This will end up with a command line with multiple `-p` but the last one (the `-p dots` in `TEST_CI_ARGS`) wins and the `--logfile` is ignored as it only applies to `-p tap`.

---

Thanks, I didn't think about that!